integration-cli/docker_cli_build_test.go

```diff
@@ -1568,7 +1569,18 @@ func TestBuildAddBadLinks(t *testing.T) {
 	}
 	defer os.RemoveAll(tempDir)
 
-	symlinkTarget := fmt.Sprintf("/../../../../../../../../../../../..%s", tempDir)
+	var symlinkTarget string
+	if runtime.GOOS == "windows" {
+		var driveLetter string
+		if abs, err := filepath.Abs(tempDir); err != nil {
+			t.Fatal(err)
+		} else {
+			driveLetter = abs[:1]
+		}
+		symlinkTarget = fmt.Sprintf(`%s:\..\..\..\..\..\..\..\..\..\..\..\..%s`, driveLetter, tempDir)
```

Comment:
Something seems wrong to me here:

1. I believe `ioutil.TempDir()` returns an absolute path, so `tempDir` is
2. Because of this, I don't think the call to `filepath.Abs` is necessary
3. In the end, I'm very surprised that `symlinkTarget` ends up being a valid path because assuming `tempDir` is absolute, you'll make up something that looks like `"C:\..\..\..C:\Users\icecrime\AppData\Local\Temp\temp123456789`"

Am I missing something?